src/test/java/com/akademia/projectplanner/mapper/TaskMapperTest.java

```diff
@@ -0,0 +1,119 @@
+package com.akademia.projectplanner.mapper;
+
+import com.akademia.projectplanner.dto.TaskDto;
+import com.akademia.projectplanner.entity.TaskEntity;
+import com.akademia.projectplanner.entity.UserEntity;
+import com.akademia.projectplanner.enums.Status;
+import com.akademia.projectplanner.exception.TaskDoesNotExistException;
+import com.akademia.projectplanner.exception.UserDoesNotExistException;
+import com.akademia.projectplanner.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskMapperTest {
+
+  private TaskDto taskDto;
+  private TaskEntity taskEntity;
+
+  @Mock private UserRepository userRepository;
+  @InjectMocks private TaskMapper taskMapper;
+
+  @BeforeEach
+  void setUp() {
+    taskDto = new TaskDto();
+    taskEntity = new TaskEntity();
+    MockitoAnnotations.openMocks(this);
```

Comment:
co prawda testy przechodzą ale jeśli chcemy używać openMocks() w sposób prawilny to powinniśmy jeszcze zamknąć mocki po każdym z testów:
https://javastart.pl/baza-wiedzy/testowanie-jednostkowe/mockito-tworzenie-mockow
pod hasłem: "Adnotacja @Mock i MockitoAnnotations.openMocks()"

osobiście nawet tego nigdy nie stosowałem tylko poprzez adnotację @ExtendWith(MockitoExtension.class),
więc fajny przykład się trafił :)